packages/apps/tools/src/pages/transactions/transfer/index.tsx

```diff
@@ -38,6 +39,28 @@ const TransferPage = () => {
   const [receiverChainId, setReceiverChainId] = useState<ChainId>(CHAINS[0]);
   const [senderChainId, setSenderChainId] = useState<ChainId>(CHAINS[0]);
 
+  const [sidebarOpen, setSidebarOpen] = useState<boolean>(false);
+
+  const helpInfoSections = [
+    {
+      question: 'What was my key index, again?',
```

Comment:
Should we have these coming from the translation file as well?

Reply:
I thought when we have the final ones, we would create a separate PR and add these in the translations files.

Reply:
Makes sense 👍